analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/SymbolicExecution/SETestContext.cs

```diff
@@ -114,8 +114,9 @@ public void Main(bool boolParameter{additionalParameters})
 
     public NotImplementedException CreateException() => new NotImplementedException();
 
-    private void Tag(string name, object arg = null) {{ }}
-    private T Unknown<T>() => default;
+    private void Tag(string name) {{ }}
```

Comment:
What's this one for?

Reply:
There are a lot of tests for try/finally that call `Tag("BeforeTry")`. We can not make this one static because of:
https://github.com/SonarSource/sonar-dotnet/blob/dae2cfd92d0656c3f54b4f7f858e1c930f0d0a0c/analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.IsNull.cs#L171-L207